src/core/css/inc/mixins.scss

```diff
@@ -219,17 +219,8 @@ $default-arrow-margin: 3px;
     font-weight: 400;
   }
 
-  // This is required because a <button> element with a :hover effect
-  // that changes its appearance will transition to the hover state onClick
-  // when using a mobile browser.
-  // This is the cause of
-  // https://github.com/mozilla/addons-frontend/issues/1904
-  //
-  // For more info, see: https://css-tricks.com/annoying-mobile-double-tap-link-issue/#article-header-id-2
-  @media (pointer: fine) {
```

Comment:
Won't we still need something like this for all other buttons? I liked how it was a generic solution that would apply globally. If it doesn't make sense for this patch, maybe just file an issue.